src/shader.cpp

```diff
@@ -374,14 +401,6 @@ void ShaderProgram::setUniforms()
     }
 }
 
-
-void ShaderProgram::setContext(const QGLContext* context)
-{
-    m_context = context;
-    setShader(shaderSource());
-}
```

Comment:
I actually changed this, because the more recent `QOpenGLShader` doesn't allow you to set the context anymore and just uses the current one available, which also makes sense. It will be easier to switch to that class at some point in the future. I didn't see any particular problem with the previous implementation, though.